QuestionBot/QuestionBot/Model/MessageEmitter.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace QuestionBot.Model {
+    public class MessageEmitter : IMessageEmitter {
+        private string _lineInput = "";
+        private static IList<IMessageListener> _listeners = new List<IMessageListener>();
+
+        public void Start() {
+            while( _lineInput != "/exitQuestionbot" ) {
+                _lineInput = Console.ReadLine();
+                NotifyAllListeners( _lineInput );
+            }
+        }
+
+        public void Add( IMessageListener listener ) {
+            _listeners.Add( listener );
+        }
+
+        public static void NotifyAllListeners( string newInput ) {
```

Comment:
I love that this logic is in its own function, but it should be `private` because we don't want consumers to be calling this method at the wrong time (also, it's not on the interface).